resources/javascript/cards.js

```diff
@@ -0,0 +1,106 @@
+let update;
+update = function () {
```

Comment:
Should be consistent style throughout, since the arrow notation is used in the same document, use `update = () => {...}`

Reply:
And are you going to change the `update` function itself throughout the script? If not, use `const update = () => {...}`, pretty sure that works fine